### PR TITLE
Improve contibutiong flow: Add issue and PR templates

### PR DESCRIPTION
**Pull request by MaxymVlasov**
[Badges and logo preview](https://github.com/shalb/cluster.dev/tree/improve_contibuting_flow#clusterdev---kubernetes-based-dev-environment-in-minutes)

Issue template will be [looks like this](https://github.com/kubernetes/website/issues/new/choose)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: Report a bug encountered with cluster.dev
```
Comment (anastyakulyk) [inline .github/ISSUE_TEMPLATE/bug-report.md:3]:
```suggestion
about: Report a bug encountered in cluster.dev
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+**Cluster.dev Version:**
+
+**Cluster.dev Config:**
+
+```yaml
+
+```
+
+
+<!--Optional Information (remove the comment tags around information you would like to include)-->
```
Comment (anastyakulyk) [inline .github/ISSUE_TEMPLATE/bug-report.md:21]:
```suggestion
<!--Optional Information (be sure to remove the comment tags around the information that you would like to include)-->
```

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+name: Feature Request
+about: Suggest a/an feature/enhancement to the cluster.dev project
```
Comment (anastyakulyk) [inline .github/ISSUE_TEMPLATE/feature-request.md:3]:
```suggestion
about: Suggest a feature/enhancement to the cluster.dev project
```

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+labels:
+- kind/feature
+---
+
+<!-- Please only use this template for submitting feature/enhancement requests -->
```
Comment (anastyakulyk) [inline .github/ISSUE_TEMPLATE/feature-request.md:8]:
```suggestion
<!-- Please use this template only for submitting feature/enhancement requests -->
```

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+**What would you like to be added**
```
Comment (anastyakulyk) [inline .github/ISSUE_TEMPLATE/feature-request.md:10]:
```suggestion
**What would you like to add**
```

File: .github/ISSUE_TEMPLATE/support.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+---
+name: Support Request
+about: Support request or question relating to cluster.dev project
```
Comment (anastyakulyk) [inline .github/ISSUE_TEMPLATE/support.md:3]:
```suggestion
about: Support request or a question relating to the cluster.dev project
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<!-- 🛈
+
+ Hello!
+
+ Remember to ADD A DESCRIPTION and delete this note before submitting
+ your pull request. The description should explain what will change,
```
Comment (anastyakulyk) [inline .github/PULL_REQUEST_TEMPLATE.md:6]:
```suggestion
 your pull request. The description should explain what will be changed,
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+ and why.
+
+ For overall help on editing and submitting pull requests, visit:
```
Comment (anastyakulyk) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
 For the detailed guidance on editing and submitting pull requests, please visit:
```

**Comment by voatsap** [comment]
Regarding badges, Please keep slack:join and twitter-follow (if possible badge without followers number).
Contributors and contribution/month - is obvious stat from first sight on repo. so lets remove them

**Pull request merged**